doc/Factory-Compatibility.md

```diff
@@ -408,7 +408,7 @@ _Note: This list is not exhaustive of all the methods defined by each factory. T
 |                         `Collection#as_binary` |  ✅   |    ✅    |  ✅  |   ✅    |     ✅     |     ✅      |     ✅     |
 |                            `Collection#empty?` |  ✅   |    ✅    |  ✅  |   ✅    |     ✅     |     ✅      |     ✅     |
 |                           `Collection#simple?` |  ✅   |    ✅    |  ✅  |   ✅    |     ❌     |     ✅      |     ❌     |
-|                          `Collection#boundary` |  ✅   |    ❌    |  ✅  |   ❌    |     ❌     |     ✅      |     ❌     |
+|                          `Collection#boundary` |  ❌   |    ❌    |  ❌  |   ❌    |     ❌     |     ❌      |     ❌     |
```

Comment:
Geos would raise when calling this method, hence now we may consider removing this line altogether?

Reply:
Do you mean remove the method from `GeometryCollection` entirely? I'm ok with that, though it might make more sense to raise `UnsupportedOperation` instead.

Reply:
I'm OK with that solution as well. I looked quickly what SFS 1.1 (https://gismanual.com/relational/99-049_OpenGIS_Simple_Features_Specification_For_SQL_Rev_1.1.pdf) has to say about it: it looks like a collection should have a boundary method, however the behaviour of such a method is not explained...

If you could have a second look on that, maybe you'll understand better

Reply:
Yeah, it seems unclear there. Seems like it's possible given a certain collection of geometries depending on the mix of them. Here's the GEOS implementation (https://github.com/libgeos/geos/blob/1dc4b60637c6d1c190c44accd2e524aef57489d7/src/geom/GeometryCollection.cpp#L198-L201) where they just throw an exception. I'd say it's probably better to align with GEOS and just raise `UnsupportedOperation` or something when we try to call it.

Reply:
I think leaving it as `InvalidGeometry` in the FFI and CAPI implementations is fine though.

Reply:
Ok then we're good to go :)